### ISSUE TITLE
[Bug]: Android - Game Crashes When Loading Local/Cloud Save

### ISSUE THREAD

**Issue by tezzy808** [MelvorIdle/melvoridle.github.io#3272]
### Describe the bug

When I launch the game on Android and select a save game (cloud or local) it starts to load but then pauses then crashes when it reaches this screen.
![Screenshot_20230324-075326](https://user-images.githubusercontent.com/51450394/227540284-845addd3-3d44-45cb-beff-3f81c1fcd621.png)

No issues playing on browser though.  This has been happening for the past two to three weeks.  I have tried clean uninstall (clear cache/storage) and install and it hasn't resolved the issue.

### Reproduction Steps

1. Launch game on Android
2. Load local/cloud save
3. Play game... but can't play because game crashes.

### Expected behaviour

I expected that I would be able to play the game like I used to :)

### Save Link

https://melvoridle.com/save/ysLcPV9D

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [ ] Mobile App (iOS)
- [X] Mobile App (Android)
- [ ] Steam
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

v1.1.1

### Game Subversion

I'm pretty sure I'm running v1.1.1 but not sure of the subversion since I can't get to the in game menu

### Are you using any mods?

No

**Comment by MelvorIdle** [comment]
The save provided works fine for me on my Oppo & Samsung Android devices. Is this issue still occuring?

**Comment by tezzy808** [comment]
The issue still persists on my Pixel 4XL.  I'll try a clean uninstall (clear cache and storage) and clean install again (I've done it twice now) and see if that helps.

Thank you for replying!
________________________________

**Comment by MelvorIdle** [comment]
Can you try leaving the game open for as long as possible? Just leave it there on the frozen screen for 10,20,30 minutes and see if it eventually loads in.

**Comment by tezzy808** [comment]
Good morning,

tl;dr... I believe the issue can be considered resolved and this ticket closed.

Longer story... on Tuesday April 4 (when you reached out to me) the problem persisted.  Then on Wednesday I was able to play via the Android app again (yay!) but then on Thursday the problem returned.  I did a clean uninstall/reinstall and the same problem persisted for the rest of Thursday.  Since Friday April 7 (I believe) I have been able to play via the Android app again!

I'm not sure if anything was done on the back-end... but, thanks!

________________________________
